requirements/common.in

```diff
@@ -135,7 +135,7 @@ py3dns
 
 # Install Python Social Auth
 social-auth-app-django
-https://github.com/python-social-auth/social-core/archive/3.4.0.zip/#egg=social-auth-core[azuread,saml]==3.4.0
+social-auth-core[azuread,saml]<4.0.3  # 4.0.3 needs PyJWT 2: https://github.com/Pr0Ger/PyAPNs2/pull/122, https://github.com/twilio/twilio-python/issues/556
```

Comment:
We may want to schedule replacing pyAPNs2 with a better library, unless we think they're likely to fix this soon.

Reply:
Twilio needs PyJWT 1 too at the moment.